pyccel/codegen/printing/fcode.py

```diff
@@ -465,7 +465,7 @@ def _print_Print(self, expr):
 
     def _print_SymbolicPrint(self, expr):
         # for every expression we will generate a print
-        code = ''.join(["print *, 'sympy> {}'".format(a)])
```

Comment:
I think that passing a generator expression to `str.join()` should be more efficient than passing a list.
You may simply remove the square brackets!